Document/0x04a-Mobile-App-Taxonomy.md

```diff
@@ -50,6 +50,6 @@ PWAs are supported by Android and iOS, but not all hardware features are yet ava
 
 ## What's Covered in the Mobile Testing Guide
 
-Throughout this guide, we will focus on apps for the two platforms dominating the market: Android and iOS. Mobile devices are currently the most common device class running these platforms – increasingly however, the same platforms (in particular, Android) run on other devices, such as smartwatches, TVs, car navigation/audio systems, and other embedded systems.
+Throughout this guide, we will focus on apps for the two platforms dominating the market: Android and iOS. Mobile devices are currently the most common device class running these platforms. However, the same platforms (or adapted versions of them) run on other devices, such as smartwatches, tablets, TVs, car navigation/audio systems, and other embedded systems.
```

Comment:
> Mobile devices are currently the most common device class running these platforms.

I found this sentence confusing, since it flips subject and object. I think this can be made more clear using something like: "These platforms commonly run on mobile devices."

> tablets

Is this also a mobile device?